### DISCUSSION TITLE
Is there any way to upgrade bootstrap classes by default in the project while migrating from Bootstrap version 4 to version 5.

### DISCUSSION THREAD

**Discussion by vidyaflorency** [twbs/bootstrap#37819] [Q&A]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

While migrating from bootstrap 4 to bootstrap 5, is there anyway to upgrade the bootstrap 4 classes to bootstrap 5 classes in existing project?

### Reduced test cases

Upgrading bootstrap classes while migrating from v4 to v5.

### What operating system(s) are you seeing the problem on?

Windows

### What version of Bootstrap are you using?

v4 migrating to v5

**Comment by tbolt** [comment]
There's no way to programmatically update the classes as far as I know. You can take a look at the [migration guide docs ](https://getbootstrap.com/docs/5.0/migration/) which may be helpful.

**Comment by vidyaflorency** [comment]
So, after upgrading the bootstrap versions the only option is to check new classes in Bootstrap v5 and update the class names manually? Is that right?

**Reply by julien-deramond** [reply]
Please be careful because Bootstrap is not only classes, it provides also HTML markup that can be modified between a version and another. As mentioned by @tbolt, the [migration guide](https://getbootstrap.com/docs/5.3/migration/) will help you. If you need at some point more precision about a specific change you can also check the [Release note](https://github.com/twbs/bootstrap/releases/tag/v5.3.0-alpha1) of the corresponding target version.
